content/shmem_atomic_compare_swap.tex

```diff
@@ -46,8 +46,7 @@
     atomic operation.
 }
 \apidesctable{
-    The  \VAR{dest} and \VAR{value} data objects must conform to certain typing
-    constraints, which are as follows:
+    When using \Fortran, \VAR{dest} and \VAR{value} must be of the following type:
```

Comment:
Should this one also include `\VAR{cond}`?

Reply:
Poke @davidozog

Reply:
Ah yes.  Sorry, I'm on vacation this week.  I've made the change.

Reply:
Thanks! Sorry to bug you on vacation!